### ISSUE TITLE
Small bug while installing

### ISSUE THREAD

**Issue by theqoobee** [RemixDev/DRemixIssues#176]
When running npm install on the provided source code, the postinstall script fails to run.
In turn building the app will then not work.
It's not a big issue, but just something I thought I'd mention.

Just get into the source code and run `npm install`
It *should* install all the requirements, and then run that custom command you guys did, postinstall, but it doesn't.

```
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! undefined postinstall: `electron-builder install-app-deps`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the undefined postinstall script.
```

 - OS: Windows 10
 - Version 1809
 - Did you build it from the repo?: Yes
 - If so which branch?: Master

https://pastebin.com/PXRR8qCh - Full log file

**Comment by RemixDev** [comment]
Do tu have git installed?

**Comment by theqoobee** [comment]
Of course, I used `git clone` to clone the repo. To clarify, it ran just fine when I executed that postinstall script manually, just the automatic way does not seem to be functioning.
